src/test/java/net/starlark/java/eval/ScriptTest.java

```diff
@@ -108,6 +108,21 @@ public Struct mutablestruct(Dict<String, Object> kwargs) throws EvalException {
     return new MutableStruct(kwargs);
   }
 
+  @StarlarkMethod(
+      name = "unsafe_shallow_freeze",
+      documented = false,
+      parameters = {
+        @Param(name = "x"),
+      }
+  )
+  public void unsafeShallowFreeze(Object object) throws EvalException {
+    if (object instanceof Mutability.Freezable) {
+      ((Mutability.Freezable) object).unsafeShallowFreeze();
+    } else {
+      throw Starlark.errorf("Object is not freezable: %s", Starlark.repr(object));
```

Comment:
Use "value" not "object" in the user interface and documentation.
Also, show its type, not its str:

errorf("value of type %s is not freezable:", Starlark.type(x))